### ISSUE TITLE
Nested Mediaqueries produce an error in CSSLint

### ISSUE THREAD

**Issue by apticknor** [CSSLint/parser-lib#118]
Run the following code through CSSLint.

```
@media (min-width: 1em) {
  @media (min-width: 33.75em) {
    body { background-color: navajoWhite; }
  }
}
```

It will produce the following errors.

```
Expected RBRACE at line 2, col 3.
```

```
Unexpected token '}' at line 5, col 1.
```

Nested mediaqueries are allowed per the spec - http://www.w3.org/TR/css3-conditional/#processing

Expected behavior is that nested @media should not produce an error.

**Comment by ideadapt** [comment]
@XhmikosR close this?

**Comment by XhmikosR** [comment]
It's still valid.

**Comment by ideadapt** [comment]
Ok, the PR is open since a month now. Who has permission to merge it?

**Comment by XhmikosR** [comment]
More like 2 years.

I do but never looked at it. I will add it in my TODO list.
